model/event/subscriptions.go

```diff
@@ -37,25 +38,27 @@ const (
 )
 
 type Subscription struct {
-	ID             bson.ObjectId `bson:"_id"`
-	Type           string        `bson:"type"`
-	Trigger        string        `bson:"trigger"`
-	Selectors      []Selector    `bson:"selectors,omitempty"`
-	RegexSelectors []Selector    `bson:"regex_selectors,omitempty"`
-	Subscriber     Subscriber    `bson:"subscriber"`
-	Owner          string        `bson:"owner"`
-	OwnerType      OwnerType     `bson:"owner_type"`
+	ID             bson.ObjectId     `bson:"_id"`
+	Type           string            `bson:"type"`
+	Trigger        string            `bson:"trigger"`
+	Selectors      []Selector        `bson:"selectors,omitempty"`
+	RegexSelectors []Selector        `bson:"regex_selectors,omitempty"`
+	Subscriber     Subscriber        `bson:"subscriber"`
+	OwnerType      OwnerType         `bson:"owner_type"`
+	Owner          string            `bson:"owner"`
+	TriggerData    map[string]string `bson:"trigger_data,omitempty"`
```

Comment:
I couldn't think of a better data structure for this, but let me know if you have ideas. Any struct that implements the trigger interface has 0 or more extra pieces of data that they take from the user. These all vary and there are/will be lots of triggers so I thought a map would be the cleanest

Reply:
what are (example) values for the key/values of this map? can we use better types? would a slice of structs be easier?

Reply:
One example is {duration_secs: 900}, but more generally it will hold any parameterized trigger conditions mentioned in this section: https://docs.google.com/document/d/1LOvRtM4cN67BDlfnS8cZ5OQ-TmJ2zSIC30CalxiUIPo/edit#heading=h.putk0ejft2vw

I don't think a slice of structs would be easier since the structs would essentially be key/value pairs